conduct.html

```diff
@@ -87,6 +87,12 @@ <h2>Scope</h2>
         account, or acting as an appointed representative at an online or
         offline event. Representation of a project may be further defined and
         clarified by project maintainers.
+
+        The project spaces under this Code of Conduct are:
+
+        * Issue trackers of the repositories under The Rails organization at GitHub (https://github.com/rails)
+        * The Rails official mailing lists. (https://groups.google.com/forum/#!forum/rubyonrails-core and https://groups.google.com/forum/#!forum/rubyonrails-security)
```

Comment:
* -official
* trailing period here but not on other list items
* rubyonrails-talk as well

Reply:
Should we change those links to https://discuss.rubyonrails.org/ ?